studio/lib/constants/metrics.tsx

```diff
@@ -81,12 +81,6 @@ export const METRICS = [
   //   provider: 'daily-stats',
   //   category: METRIC_CATEGORIES.API_REALTIME,
   // },
-  {
-    key: 'total_realtime_requests',
-    label: 'Connection Requests',
```

Comment:
This is a duplicate of https://github.com/supabase/supabase/pull/9871/files#diff-35531569daebd9fa0bc01611016cc9e705a5dd94ef4f5c4742c1e50493b200e2R68.